MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: post
```

Comment:
title: Data annotations in .NET MAUI DataForm (SfDataForm) control

Learn here all about annotations support in Syncfusion .NET MAUI DataForm (SfDataForm) control to handle the data.